### PR TITLE
VisualizationSuggestions: Support image & text instead of real previews. Adds suggestions for all non data panels when there are no data

### PR DESCRIPTION
**Pull request by torkelo**
Some times real previews of the panel are not that useful, for example for nondata panels, panels like dashlist & alert list & news panel etc. And in case there is no data at all but we still want to present a table as a good suggestion (even when there is no data). 

* [x] Adds cardOptions to VisualizationSuggestion so a suggestion can be shown with just an image and text instead of real preview
* [x] Use this new card option for table when there is no data 
* [x] Add suggestions for all "skipDataQuery" panels when there is no data 

Before: 
Screenshot from 2021-11-22 14-41-31

After:
Screenshot from 2021-11-22 14-37-30

### CODE REVIEW

**Review by dprokop — APPROVED** [review]
Looks good!

File: public/app/features/panel/components/VizTypePicker/VisualizationSuggestionCard.tsx [hunk 1]
```diff
@@ -88,13 +96,34 @@ const getStyles = (theme: GrafanaTheme2) => {
...
       overflow: hidden;
       color: ${theme.colors.text.secondary};
       font-weight: ${theme.typography.fontWeightMedium};
       text-overflow: ellipsis;
+      padding-bottom: ${theme.spacing(0.5)};
+      margin-top: ${theme.spacing(-2)};
+    `,
+    img: css`
+      max-width: 65px;
+      max-height: 65px;
```
Comment (ashharrison90) [inline public/app/features/panel/components/VizTypePicker/VisualizationSuggestionCard.tsx:126]: should this go in the theme somewhere or can we use a theme value here? either `theme.components.img.sm/md/lg` or `theme.spacing(8)`?

File: public/app/features/panel/components/VizTypePicker/VisualizationSuggestionCard.tsx [hunk 2]
```diff
@@ -88,13 +96,34 @@ const getStyles = (theme: GrafanaTheme2) => {
...
     name: css`
+      padding: ${theme.spacing(1)};
       font-size: ${theme.typography.bodySmall.fontSize};
       white-space: nowrap;
       overflow: hidden;
       color: ${theme.colors.text.secondary};
       font-weight: ${theme.typography.fontWeightMedium};
       text-overflow: ellipsis;
+      padding-bottom: ${theme.spacing(0.5)};
+      margin-top: ${theme.spacing(-2)};
```
Comment (ashharrison90) [inline public/app/features/panel/components/VizTypePicker/VisualizationSuggestionCard.tsx:122]: i'm not a huge fan of negative margins tbh. if the duplicate `padding` on 114 is removed everything looks fine to me without it. wdyt?

**Review by ashharrison90 — APPROVED** [review]
looks great! 🙌 

couple of minor css nits, maybe personal preference 🤷

File: public/app/features/panel/components/VizTypePicker/VisualizationSuggestionCard.tsx (hunk 2 above)
Comment (torkelo) [reply public/app/features/panel/components/VizTypePicker/VisualizationSuggestionCard.tsx:122]: me neither but what you suggest is not the same, want the image to be perfectly centered and it's not unless I push it back up to account for the space the heading/name text takes

**Pull request merged**